packages/riipen-ui/src/components/Breadcrumbs.test.jsx

```diff
@@ -0,0 +1,189 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Breadcrumbs from "./Breadcrumbs.jsx";
+
+describe("<Breadcrumbs>", () => {
+  it("renders without errors", () => {
+    let error;
+    const child = <h1>hello hello</h1>;
+    try {
+      mount(<Breadcrumbs children={child} />);
+    } catch (e) {
+      error = e;
+    }
+    expect(error).toEqual(undefined);
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const child = <h1>hello hello</h1>;
+      const defaultProps = new Breadcrumbs({ children: child }).type
+        .defaultProps;
+
+      const wrapper = mount(<Breadcrumbs children={child} />);
+
+      const component = wrapper.find("Breadcrumbs");
+      expect(component.props().component).toEqual(defaultProps.component);
+      expect(component.props().separator).toEqual(defaultProps.separator);
+
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+  });
+
+  describe("children prop", () => {
+    it("renders children nodes as typography", () => {
```

Comment:
I think this test is actually testing that the children are rendered as a <li> not as <Typography>. We should up this title accordingly.